### ISSUE TITLE
VSCode: Opening up protein fasta files is really inconsistent

### ISSUE THREAD

**Issue by KeaunAmani** [bioSyntax/bioSyntax#66]
Opening up a fasta file in vscode with the fasta-taylor language mode results in this at start:
![image](https://user-images.githubusercontent.com/108769308/229213294-1565efa4-5061-4a51-a9ba-ad54b7ac323b.png)

But then after switching in and out of the tab I get this:
![image](https://user-images.githubusercontent.com/108769308/229213367-65ec0aff-1ff9-48a9-9dc9-58ed775a4c11.png)

Both cases seem to be bugged as in the first case you have the letter highlighting instead of changing the text color. The highlighting is annoying as it makes selecting text unnecessarily difficult. Furthermore, the second screenshot there's no text coloring at all which is unfortunate.

I looked through the extension settings but didn't see anything useful.

**Comment by liyao001** [comment]
This looks weird. Can you post the entire file here (including the extension name)? I can take a look at this later.

**Comment by KeaunAmani** [comment]
@liyao001 It seems to happen with all files that end with the .fasta extension, including this one I just downloaded from the pdb:
![image](https://user-images.githubusercontent.com/108769308/229264625-1d47873e-ee7f-4b49-a7db-b30c05b7ac50.png)

Note that I added the .txt extension just for uploading to github, but I opened the file as `rcsb_pdb_7N0R.fasta`
[rcsb_pdb_7N0R.fasta.txt](https://github.com/bioSyntax/bioSyntax/files/11128401/rcsb_pdb_7N0R.fasta.txt)

**Comment by liyao001** [comment]
This issue happens because the plugin is confused about whether the sequences are for DNA or protein. I'll think about possible ways to fix this. Let's keep this issue open for now.

**Comment by KeaunAmani** [comment]
Yeah sounds good. I think this is a pretty solid plugin, it's just a tad unfortunate that the fasta parsing isn't quite usable yet.

**Comment by ababaian** [comment]
can you rename your fast file extension to `.faa` and try again?

**Comment by KeaunAmani** [comment]
@ababaian it just looks like this
![image](https://user-images.githubusercontent.com/108769308/230199207-e95bba14-4f25-4d49-99d6-07af266ee380.png)
